src/symmetric.jl

```diff
@@ -987,6 +988,7 @@ See also [`hermitianpart!`](@ref) for the corresponding in-place operation.
     This function requires Julia 1.10 or later.
 """
 hermitianpart(A::AbstractMatrix, uplo::Symbol=:U) = Hermitian(_hermitianpart(A), uplo)
+hermitianpart(x::Number) = float(real(x))
```

Comment:
```suggestion
hermitianpart(x::Number) = real(x)
```
I don't think `float` is necessary?

Reply:
As per the discussion above, this should be the right solution. Sorry for the back and forth.

Reply:
I'll proceed with the modification!

For informational purposes:
real(x) was chosen for its advantages in preserving precision and avoiding unnecessary type conversions, rather than float(real(x)), which would force a conversion to Float64 and risk losing precision for rational numbers?

Thanks